.pre-commit-config.yaml

```diff
@@ -18,8 +18,8 @@
       files: molecule/.*/create\.yml
       args: [-x, ANSIBLE0016]
 
-- repo: git://github.com/pre-commit/pre-commit-hooks
-  sha: v0.9.1
+- repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
there's no .git appended to this https url, I don't think its necessary but we are using it above ^